cli/tests/pcluster/config/test_config_patch.py

```diff
@@ -76,8 +77,19 @@ def _compare_change(source, target):
             and source.is_list == target.is_list
         )
 
-    sorted_changes = sorted(changes, key=lambda change: change.key)
-    sorted_expected_changes = sorted(expected_changes, key=lambda change: change.key)
+    def _sorting_func(change):
+        if change.is_list:
+            if isinstance(change.new_value, dict):
+                return change.new_value["Name"]
+            else:
+                return "-"
+        else:
+            return change.key
+
+        return change.new_value["Name"] if change.new_value else "-" if change.is_list else change.key
```

Comment:
If I'm not wrong this code is never executed.

Reply:
removed